scripts/validation/validate.js

```diff
@@ -178,13 +166,9 @@ function assertValid(schema, subject) {
 }
 
 async function getModifiedServices() {
-  const git = simpleGit(path.resolve(__dirname, '../..'), { maxConcurrentProcesses: 1 });
-  const modifiedFilesPathString = await git.diff([ '--name-only', 'master...', 'services/*.json' ]);
-
-  if (!modifiedFilesPathString) {
-    return;
-  }
+  const git = simpleGit(rootPath, { maxConcurrentProcesses: 1 });
+  const modifiedFilesPathString = await git.diff([ '--name-only', 'master', 'services/*.json' ]);
```

Comment:
From my tests, if your remove `...` it will try to validate services which are modified on master but not necessary present on the current branch and it will fails in these cases.
So it is necessary. See https://stackoverflow.com/a/7256391